client/src/app/main/dashboard/dashboard-article/dashboard-article-page.js

```diff
@@ -76,9 +76,9 @@ class DashboardArticlePage extends React.Component {
     getClass() {
         let classes = {
             'dashboard-article-page': true,
-            'dashboard-article-page_wrapped': _.startsWith(this.props.location.pathname, '/article/')
+            'dashboard-article-page_wrapped': _.startsWith(this.props.location.pathname, '/dashboard/article/')
```

Comment:
I believe the previous line was buggy, since that condition would never be true, `this.props.location.pathname` always returned the absolute path, starting with dashboard.
I fixed it this way but probably there's a better way (I wouldn't like this to break if we later decide to change the name of "dashboard").